### DISCUSSION TITLE
Display the number of classes

### DISCUSSION THREAD

**Discussion by pjdervan** [ultralytics/ultralytics#4905] [Q&A]
Hi All,

I would like yolov8 to display the sum of each of the classes in an image on the CLI.

In yolov7 for example, when I run inference on a custom data set it displays something like this:

12 capacitor-sam2s, 5 capacitor-mur1s, 5 capacitor-mur2s, 1 rfid, 1 ntc, 2 resistor-packs, Done. (1513.7ms) Inference, (55.4ms) NMS

How do I do this with yolov8?

**Comment by glenn-jocher** [comment]
@pjdervan hello!

In YOLOv8, the summary of detected classes and their counts is displayed by default in the CLI output after running inference. If you're using the `predict` mode, you should see a similar output to what you've described from YOLOv7, where each class and its count are listed followed by the inference and NMS times.

If you're not seeing this information, please ensure you're using the latest version of YOLOv8 and that your custom dataset is properly configured with the correct class names. The output format is designed to be informative and should provide you with the class summary without additional configuration.

For more detailed information on the `predict` mode and its outputs, you can refer to the Predict section in our documentation. If you encounter any issues or have further questions, feel free to raise them in the issues section of the repo, and we'll be glad to assist you. 😊

Happy detecting!
